### PR TITLE
Iconoclast + Matthian Miracles

### PR DESCRIPTION
**Pull request by Onutsio**
## About The Pull Request

Adds in the following: Matthian Miracles and a new bandit cleric class(ICONOCLAST)

MATTHIAN MIRACLES

T0 - Appraise - Checks the targets' mammon-on-person value, and their meister value, tells the caster both silently.

T1 - TRANSACT - Trade a item of value(minimum 10) in your hand for a miracle heal which scales off the value of the item you are holding. Rather powerful with high value items, but completely destroys the item being transacted. The value on this probably needs testing. Very high value items are incredible sources of healing, but of course, it has the cost of just Eating it.

T2 - EQUALIZE - Check a target's stats, if they have net stats higher than us(by atleast 4 points, weighing strength as double). We half our total difference(to a max of +2/-2, and apply it as a debuff to the targets' and a buff to our own for a minute. 

T3 - Churn Wealthy - Title rather explanatory. Checks the net mammon value of a target(adding +101 if they have nobility status, automatically putting nobles at the mid level). Deals scaling damage/effects based on net value. Over 500, and it old-style churns the target. I would roughly equivocate the 'middle' wealth smite as being like the old miracle burns vs undead, for reference.

ICONOCLAST

_"Trained by an Ecclesial sect, you uphold the Ideological purity of the Matthian Creed. Take from the wealthy, give to the worthless, empower. They will look up to you, in search for the God of Robbery's guidance. Be their light in the dark."_

A Matthian Cleric, trained by some Ecclesial Sect and sent to support more fringe assets and followers of the Robber-God. This single slot bandit class boasts expert unarmed skills(and jman poles/maces,  if you are a wimp), heavy armor, and the new Matthian miracle list.

...Also fixes some Hoardmaster supply pack typos, which I will likely be doing for all eternity.

EDIT:

Also enables ascendant paladins, grants battlemasters shield skill(they start with one), and makes it so that sacks over people's heads prevent them from learning travel tile paths(hostages leading armies back to the bandit camp despite having been sacked)

## Why It's Good For The Game

I lomve.. Miracles.. I lomve... unarmed.. I lomve... the rp potential of a bandit ideological guide.

### CODE REVIEW

**Comment by Onutsio** [comment]
Enabled Ascendant Paladins while Im Touching ;3

**Comment by Cleric2145** [comment]
Churn the wealthy is the new battlecry for 2025

**Comment by Ruediger4** [comment]
> Enabled Ascendant Paladins while Im Touching ;3

How far would they get? T2?

**Comment by Onutsio** [comment]
> > Enabled Ascendant Paladins while Im Touching ;3
> 
> How far would they get? T2?

T1! So Matthian Paladins would get Transact and Appraise.

**Comment by Onutsio** [comment]
> I made this quip on Discord, but;
> 
> Churn wealthy by it's nature can be exploited by (really obvious) power gamers. Your meister balance is independent of the treasury, so in theory someone can just run around and give everyone like 5000 mammon by class. And sure, someone could empty out the treasury, but you still have that bank balance. So there is the case of bandits breaking in to give everyone money, or at least a target money, and then airstriking them off the map.

I think this is a hilarious thought experiment but dont think It'll realistically happen. Its alot of effort esp with the new alarms for a antag that can also just beat their target to death

**Comment by radarseas** [comment]
it would be funny if the `"burned with holy light"` for over 500 wealth was changed to `"smited with holy light"` to empathise the complete noble obliteration this is casting, _a forced scream on the victim would be even funnier_.

**Comment by mopcat** [comment]
> I made this quip on Discord, but;
> 
> Churn wealthy by it's nature can be exploited by (really obvious) power gamers. Your meister balance is independent of the treasury, so in theory someone can just run around and give everyone like 5000 mammon by class. And sure, someone could empty out the treasury, but you still have that bank balance. So there is the case of bandits breaking in to give everyone money, or at least a target money, and then airstriking them off the map.

most bandits would, if given the sort of access that would enable them to do this, take all the money and leave to go offer it to the idol. there is no practical reason to do this other than killing a specific person in spectacular fashion.

File: modular_azurepeak/code/modules/spells/pantheon/inhumen/matthios.dm [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+			user.STAEND += totalstatshift
+			target.STALUC -= totalstatshift
+			user.STALUC += totalstatshift
+			target.STAINT -= totalstatshift
+			user.STAINT += totalstatshift
+			target.STACON -= totalstatshift
+			user.STACON += totalstatshift
+			target.STAPER -= totalstatshift
+			user.STAPER += totalstatshift
+			spawn(1 MINUTES) // timer before our stats shift back to normal.
```
[the remark this answers is not in the document]
Comment (Onutsio) [reply modular_azurepeak/code/modules/spells/pantheon/inhumen/matthios.dm:138]: I knew someone would tell me how to make that better. Will look when at computer

[the remark this answers is not in the document]
Comment (Onutsio) [reply modular_azurepeak/code/modules/spells/pantheon/inhumen/matthios.dm]: Resolved with timers as advised. Thank u for ur help macha awawawa <3

**Comment by Onutsio** [comment]
Unarmed skill bumped to master(Being their intended, and only fighting skill). Runtime machine resolved. Good for Full merge!

**Pull request merged**